nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/CreateHadoopSequenceFile.java

```diff
@@ -64,7 +64,8 @@
 @SideEffectFree
 @InputRequirement(Requirement.INPUT_REQUIRED)
 @Tags({"hadoop", "sequence file", "create", "sequencefile"})
-@CapabilityDescription("Creates Hadoop Sequence Files from incoming flow files")
+@CapabilityDescription("Creates Hadoop Sequence Files from incoming flow files."
+        + " If you want to use SSL-secured file system like swebhdfs, please see the 'SSL Configuration' topic of the 'Additional Details' of PutHDFS.")
```

Comment:
I think we should avoid adding the 'if you want to use SSL-Secured file system" entry in all these descriptions.  If it is an option of the processor then on that option this comment can exist.

Reply:
@joewitt I agree, the explanation is not a 'capability description' rather a documentation for configuration detail.

Is there any better place to write a common topic like this for components within a NAR package? Something like 'package.html' in Javadoc, describing what the NAR includes and common configuration, what type of flow user can construct with provided processors ... etc. Is there any improvement going on in 'Extension Registry' sub project for such documentation? If not, does it sounds helpful? If so, I will raise a JIRA to add 'bundle.html' or 'nar.html' (Probably with Markdown support). How do you think?